test_conformance/images/kernel_read_write/test_common.cpp

```diff
@@ -1543,4 +1543,39 @@ int test_read_image(cl_context context, cl_command_queue queue,
     }
 
     return numTries != MAX_TRIES || numClamped != MAX_CLAMPED;
-}
\ No newline at end of file
+}
+
+void filter_undefined_bits(image_descriptor *imageInfo, char *resultPtr)
+{
+    // mask off the top bit (bit 15) if the image format is (CL_UNORM_SHORT_555,
+    // CL_RGB). (Note: OpenCL says: the top bit is undefined meaning it can be
+    // either 0 or 1.)
+    if (imageInfo->format->image_channel_data_type == CL_UNORM_SHORT_555)
+    {
+        cl_ushort *temp = (cl_ushort *)resultPtr;
+        temp[0] &= 0x7fff;
```

Comment:
Apologies for asking after this has been merged, but should this have also dealt with CL_UNORM_INT_101010?